mediasoup doesn't compile on Alpine Linux 3.23 due GCC being upgraded from 14 to 15

# Bug Report
## Your environment

- Operating system: alpine 3.23
- gcc/clang version: 15
- mediasoup (Node) version: 3.14.14 / 3.19.13

## Issue description

I am using mediasoup in node.js with Docker. I am using Alpine image. I used 3.14.14, now updated to 3.19.13. Couple days ago I found a compilation error (below). I fixed it when I switched from Alpine to Debian image. Next, I found that if I switch to Alpine 3.22 (previous version), problem goes away. So, I presume couple days ago [Alpine 3.23 was released](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.23.0) where developers [upgraded GCC from 14 to 15](https://gcc.gnu.org/gcc-15/porting_to.html#header-dep-changes) and some code needs to be ported to it.

From porting guide:

> [Header dependency changes](https://gcc.gnu.org/gcc-15/porting_to.html#header-dep-changes)
> Some C++ Standard Library headers have been changed to no longer include other headers that were being used internally by the library. As such, C++ programs that used standard library components without including the right headers will no longer compile.
> 
> In particular, the following headers are used less widely within libstdc++ and may need to be included explicitly when compiling with GCC 15:
> 
> **<stdint.h> (for int8_t, int32_t etc.) and <cstdint> (for std::int8_t, std::int32_t etc.)**
> <ostream> (for std::endl, std::flush etc.)

In error log I found this:

> ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:46:33: error: 'int64_t' has not been declared
> ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:47:33: error: 'int64_t' has not been declared
> ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:47:33: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'

My broken Dockerfile:
```dockerfile
FROM node:24-alpine AS build-server # node:24-alpine image uses latest alpine which is 3.23 now

# copy some code
WORKDIR /shared
COPY ../shared/package*.json /shared
RUN --mount=type=cache,target=/root/.npm \
    npm ci
WORKDIR /server
COPY ./server/package*.json /server

# mediasoup build deps from here: https://github.com/versatica/mediasoup/blob/v3/worker/Dockerfile.alpine
RUN apk update && apk add --no-cache gcc g++ make python3 py3-pip linux-headers && rm -rf /var/cache/apk/*
RUN --mount=type=cache,target=/root/.npm \
    npm ci
```

Compilation error on Alpine 3.23:
> [server build-server 8/8] RUN --mount=type=cache,target=/root/.npm     npm ci:
58.33 npm error code 1
58.33 npm error path /server/node_modules/mediasoup
58.33 npm error command failed
58.33 npm error command sh -c node npm-scripts.mjs postinstall
58.35 npm error npm-scripts.mjs [INFO] [postinstall]
58.35 npm error npm-scripts.mjs [INFO] [postinstall] getWorkerPrebuildTarName() [workerPrebuildTarName:mediasoup-worker-3.19.13-linux-arm64-kernel6.tgz]
58.35 npm error npm-scripts.mjs [INFO] [postinstall] downloadPrebuiltWorker() [workerPrebuildTarUrl:https://github.com/versatica/mediasoup/releases/download/3.19.13/mediasoup-worker-3.19.13-linux-arm64-kernel6.tgz]
58.35 npm error npm-scripts.mjs [INFO] [postinstall] ensureDir() [dir:worker/prebuild]
58.35 npm error npm-scripts.mjs [INFO] [postinstall] ensureDir() [dir:worker/out/Release]
58.35 npm error npm-scripts.mjs [INFO] [postinstall] downloadPrebuiltWorker() | got mediasoup-worker prebuilt binary
58.35 npm error npm-scripts.mjs [INFO] [postinstall] downloadPrebuiltWorker() | checking fetched mediasoup-worker prebuilt binary in current host
58.35 npm error npm-scripts.mjs [INFO] [postinstall] couldn't fetch any mediasoup-worker prebuilt binary, building it locally
58.35 npm error npm-scripts.mjs [INFO] [postinstall] buildWorker()
58.35 npm error npm-scripts.mjs [INFO] [postinstall] installInvoke()
58.35 npm error npm-scripts.mjs [INFO] [postinstall] executeCmd(): "python3" -m pip install --upgrade --no-user --target "/server/node_modules/mediasoup/worker/pip_invoke" invoke
58.35 npm error Collecting invoke
58.35 npm error   Downloading invoke-2.2.1-py3-none-any.whl.metadata (3.3 kB)
58.35 npm error Downloading invoke-2.2.1-py3-none-any.whl (160 kB)
58.35 npm error Installing collected packages: invoke
58.35 npm error Successfully installed invoke-2.2.1
58.35 npm error npm-scripts.mjs [INFO] [postinstall] executeCmd(): "python3" -m invoke -r worker mediasoup-worker
58.35 npm error "/usr/bin/python3" -m pip install --upgrade --no-user --target "/server/node_modules/mediasoup/worker/out/pip_meson_ninja" pip setuptools
58.35 npm error Collecting pip
58.35 npm error   Downloading pip-25.3-py3-none-any.whl.metadata (4.7 kB)
58.35 npm error Collecting setuptools
58.35 npm error   Downloading setuptools-80.9.0-py3-none-any.whl.metadata (6.6 kB)
58.35 npm error Downloading pip-25.3-py3-none-any.whl (1.8 MB)
58.35 npm error    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 13.0 MB/s eta 0:00:00
58.35 npm error Downloading setuptools-80.9.0-py3-none-any.whl (1.2 MB)
58.35 npm error    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 23.7 MB/s eta 0:00:00
58.35 npm error Installing collected packages: setuptools, pip
58.35 npm error
58.35 npm error Successfully installed pip-25.3 setuptools-80.9.0
58.35 npm error "/usr/bin/python3" -m pip install --upgrade --no-user --target "/server/node_modules/mediasoup/worker/out/pip_meson_ninja"  meson==1.9.1 ninja==1.10.2.4
58.35 npm error Collecting meson==1.9.1
58.35 npm error   Downloading meson-1.9.1-py3-none-any.whl.metadata (1.8 kB)
58.35 npm error Collecting ninja==1.10.2.4
58.35 npm error   Downloading ninja-1.10.2.4-py2.py3-none-musllinux_1_1_aarch64.whl.metadata (5.3 kB)
58.35 npm error Downloading meson-1.9.1-py3-none-any.whl (1.0 MB)
58.35 npm error    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 9.6 MB/s  0:00:00
58.35 npm error Downloading ninja-1.10.2.4-py2.py3-none-musllinux_1_1_aarch64.whl (654 kB)
58.35 npm error    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 654.9/654.9 kB 20.5 MB/s  0:00:00
58.35 npm error Installing collected packages: ninja, meson
58.35 npm error
58.35 npm error Successfully installed meson-1.9.1 ninja-1.10.2.4
58.35 npm error "/server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/meson" setup --prefix "/server/node_modules/mediasoup/worker/out/Release" --bindir "" --libdir "" --buildtype release -Db_ndebug=true  "/server/node_modules/mediasoup/worker/out/Release/build"
58.35 npm error The Meson build system
58.35 npm error Version: 1.9.1
58.35 npm error Source dir: /server/node_modules/mediasoup/worker
58.35 npm error Build dir: /server/node_modules/mediasoup/worker/out/Release/build
58.35 npm error Build type: native build
58.35 npm error Project name: mediasoup-worker
58.35 npm error Project version: undefined
58.35 npm error C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error C++ compiler for the host machine: c++ (gcc 15.2.0 "c++ (Alpine 15.2.0) 15.2.0")
58.35 npm error C++ linker for the host machine: c++ ld.bfd 2.45.1
58.35 npm error Host machine cpu family: aarch64
58.35 npm error Host machine cpu: aarch64
58.35 npm error Checking for function "strtoull_l" : NO
58.35 npm error Downloading libuv source from https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
58.35 npm error Downloading libuv patch from https://wrapdb.mesonbuild.com/v2/libuv_1.51.0-1/get_patch
58.35 npm error
58.35 npm error Executing subproject libuv
58.35 npm error
58.35 npm error libuv| Project name: libuv
58.35 npm error libuv| Project version: 1.51.0
58.35 npm error libuv| C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error libuv| C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error libuv| Fetching value of define "__QNX__" : (undefined)
58.35 npm error libuv| Fetching value of define "_AIX" : (undefined)
58.35 npm error libuv| Message: System: linux
58.35 npm error libuv| Compiler for C supports arguments -fno-strict-aliasing: YES
58.35 npm error libuv| Run-time dependency threads found: YES
58.35 npm error libuv| Library dl found: YES
58.35 npm error libuv| Library rt found: YES
58.35 npm error libuv| Library m found: YES
58.35 npm error libuv| Did not find pkg-config by name 'pkg-config'
58.35 npm error libuv| Found pkg-config: NO
58.35 npm error libuv| Build targets in project: 1
58.35 npm error libuv| Subproject libuv finished.
58.35 npm error
58.35 npm error Downloading openssl source from https://www.openssl.org/source/openssl-3.0.8.tar.gz
58.35 npm error Downloading openssl patch from https://wrapdb.mesonbuild.com/v2/openssl_3.0.8-3/get_patch
58.35 npm error
58.35 npm error Executing subproject openssl
58.35 npm error
58.35 npm error openssl| Project name: openssl
58.35 npm error openssl| Project version: 3.0.8
58.35 npm error openssl| C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error openssl| C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error openssl| Dependency threads found: YES unknown (cached)
58.35 npm error openssl| Program as found: YES (/usr/bin/as)
58.35 npm error openssl| Message: OpenSSL is configured with ASM support
58.35 npm error openssl| Library dl found: YES
58.35 npm error openssl| Build targets in project: 4
58.35 npm error openssl| Subproject openssl finished.
58.35 npm error
58.35 npm error Downloading libsrtp3 source from https://github.com/versatica/libsrtp/archive/v3.0.0-beta.zip
58.35 npm error Downloading file of unknown size.
58.35 npm error
58.35 npm error Executing subproject libsrtp3
58.35 npm error
58.35 npm error libsrtp3| Project name: libsrtp3
58.35 npm error libsrtp3| Project version: 3.0.0
58.35 npm error libsrtp3| C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error libsrtp3| C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error libsrtp3| Has header "arpa/inet.h" : YES
58.35 npm error libsrtp3| Has header "byteswap.h" : YES
58.35 npm error libsrtp3| Has header "inttypes.h" : YES
58.35 npm error libsrtp3| Has header "machine/types.h" : NO
58.35 npm error libsrtp3| Has header "netinet/in.h" : YES
58.35 npm error libsrtp3| Has header "stdint.h" : YES
58.35 npm error libsrtp3| Has header "stdlib.h" : YES
58.35 npm error libsrtp3| Has header "sys/int_types.h" : NO
58.35 npm error libsrtp3| Has header "sys/socket.h" : YES
58.35 npm error libsrtp3| Has header "sys/types.h" : YES
58.35 npm error libsrtp3| Has header "sys/uio.h" : YES
58.35 npm error libsrtp3| Has header "unistd.h" : YES
58.35 npm error libsrtp3| Checking for function "sigaction" : YES
58.35 npm error libsrtp3| Checking for function "inet_aton" : YES
58.35 npm error libsrtp3| Checking for function "inet_pton" : YES
58.35 npm error libsrtp3| Checking for function "usleep" : YES
58.35 npm error libsrtp3| Checking for function "socket" : YES
58.35 npm error libsrtp3| Has header "stdint.h" : YES (cached)
58.35 npm error libsrtp3| Checking for type "size_t" : YES
58.35 npm error libsrtp3| Checking for type "unsigned long" : YES
58.35 npm error libsrtp3| Checking for size of "unsigned long" : 8
58.35 npm error libsrtp3| Checking for type "unsigned long long" : YES
58.35 npm error libsrtp3| Checking for size of "unsigned long long" : 8
58.35 npm error libsrtp3| Checking if "inline keyword check" compiles: YES
58.35 npm error libsrtp3| Dependency openssl from subproject subprojects/openssl-3.0.8 found: YES 3.0.8
58.35 npm error libsrtp3| Configuring config.h using configuration
58.35 npm error libsrtp3| Compiler for C supports arguments -Wstrict-prototypes: YES
58.35 npm error libsrtp3| Compiler for C supports arguments -funroll-loops: YES
58.35 npm error libsrtp3| Program doxygen found: NO
58.35 npm error libsrtp3| Build targets in project: 5
58.35 npm error libsrtp3| Subproject libsrtp3 finished.
58.35 npm error
58.35 npm error Downloading usrsctp source from https://github.com/sctplab/usrsctp/archive/d45b53f5dfa79533f5c5e7aefa5d7570405afb39.zip
58.35 npm error
58.35 npm error Executing subproject usrsctp
58.35 npm error
58.35 npm error usrsctp| Project name: usrsctplib
58.35 npm error usrsctp| Project version: 0.9.5.0
58.35 npm error usrsctp| C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error usrsctp| C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error usrsctp| Compiler for C supports arguments -pedantic: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wfloat-equal: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wshadow: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wpointer-arith: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Winit-self: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wno-unused-function: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wno-unused-parameter: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wno-unreachable-code: YES
58.35 npm error usrsctp| Compiler for C supports arguments -Wstrict-prototypes: YES (cached)
58.35 npm error usrsctp| Dependency threads found: YES unknown (cached)
58.35 npm error usrsctp| Has header "sys/queue.h" : NO
58.35 npm error usrsctp| Has header "sys/socket.h" : YES
58.35 npm error usrsctp| Has header "linux/if_addr.h" : YES
58.35 npm error usrsctp| Has header "linux/rtnetlink.h" : YES
58.35 npm error usrsctp| Has header "sys/types.h" : YES
58.35 npm error usrsctp| Has header "netinet/in.h" : YES
58.35 npm error usrsctp| Has header "netinet/ip.h" : YES
58.35 npm error usrsctp| Has header "netinet/ip_icmp.h" : YES
58.35 npm error usrsctp| Has header "net/route.h" : YES
58.35 npm error usrsctp| Has header "stdatomic.h" : YES
58.35 npm error usrsctp| Checking whether type "struct sockaddr" has member "sa_len" : NO
58.35 npm error usrsctp| Checking whether type "struct sockaddr_in" has member "sin_len" : NO
58.35 npm error usrsctp| Checking whether type "struct sockaddr_in6" has member "sin6_len" : NO
58.35 npm error usrsctp| Checking whether type "struct sockaddr_conn" has member "sconn_len" : NO
58.35 npm error usrsctp| Build targets in project: 6
58.35 npm error usrsctp| Subproject usrsctp finished.
58.35 npm error
58.35 npm error Downloading abseil-cpp source from https://github.com/abseil/abseil-cpp/releases/download/20240722.0/abseil-cpp-20240722.0.tar.gz
58.35 npm error Downloading abseil-cpp patch from https://wrapdb.mesonbuild.com/v2/abseil-cpp_20240722.0-4/get_patch
58.35 npm error
58.35 npm error Executing subproject abseil-cpp
58.35 npm error
58.35 npm error abseil-cpp| Project name: abseil-cpp
58.35 npm error abseil-cpp| Project version: 20240722.0
58.35 npm error abseil-cpp| C++ compiler for the host machine: c++ (gcc 15.2.0 "c++ (Alpine 15.2.0) 15.2.0")
58.35 npm error abseil-cpp| C++ linker for the host machine: c++ ld.bfd 2.45.1
58.35 npm error abseil-cpp| Compiler for C++ supports arguments /DNOMINMAX: NO
58.35 npm error abseil-cpp| Compiler for C++ supports arguments -Wno-sign-compare: YES
58.35 npm error abseil-cpp| Compiler for C++ supports arguments -Wno-gcc-compat: NO
58.35 npm error abseil-cpp| Checking for size of "void*" : 8
58.35 npm error abseil-cpp| Compiler for C++ supports arguments -march=armv8-a+crypto: YES
58.35 npm error abseil-cpp| Checking if "atomic builtins" links: YES
58.35 npm error abseil-cpp| Dependency threads found: YES unknown (cached)
58.35 npm error abseil-cpp| Run-time dependency appleframeworks found: NO (tried framework)
58.35 npm error abseil-cpp| Build targets in project: 21
58.35 npm error abseil-cpp| Subproject abseil-cpp finished.
58.35 npm error
58.35 npm error Downloading flatbuffers source from https://github.com/google/flatbuffers/archive/v24.3.25.tar.gz
58.35 npm error Downloading flatbuffers patch from https://wrapdb.mesonbuild.com/v2/flatbuffers_24.3.25-1/get_patch
58.35 npm error
58.35 npm error Executing subproject flatbuffers
58.35 npm error
58.35 npm error flatbuffers| Project name: flatbuffers
58.35 npm error flatbuffers| Project version: 24.3.25
58.35 npm error flatbuffers| C++ compiler for the host machine: c++ (gcc 15.2.0 "c++ (Alpine 15.2.0) 15.2.0")
58.35 npm error flatbuffers| C++ linker for the host machine: c++ ld.bfd 2.45.1
58.35 npm error flatbuffers| Checking for function "strtoull_l" : NO (cached)
58.35 npm error flatbuffers| Compiler for C++ supports arguments -fsigned-char: YES
58.35 npm error flatbuffers| Build targets in project: 31
58.35 npm error flatbuffers| Subproject flatbuffers finished.
58.35 npm error
58.35 npm error Downloading catch2 source from https://github.com/catchorg/Catch2/archive/v3.8.1.tar.gz
58.35 npm error
58.35 npm error Executing subproject catch2
58.35 npm error
58.35 npm error catch2| Project name: catch2
58.35 npm error catch2| Project version: 3.8.1
58.35 npm error catch2| C++ compiler for the host machine: c++ (gcc 15.2.0 "c++ (Alpine 15.2.0) 15.2.0")
58.35 npm error catch2| C++ linker for the host machine: c++ ld.bfd 2.45.1
58.35 npm error catch2| Configuring catch_user_config.hpp using configuration
58.35 npm error catch2| Build targets in project: 34
58.35 npm error catch2| Subproject catch2 finished.
58.35 npm error
58.35 npm error Program flatc found: YES (overridden)
58.35 npm error Downloading liburing source from https://github.com/axboe/liburing/archive/refs/tags/liburing-2.5.tar.gz
58.35 npm error Downloading liburing patch from https://wrapdb.mesonbuild.com/v2/liburing_2.5-2/get_patch
58.35 npm error
58.35 npm error Executing subproject liburing
58.35 npm error
58.35 npm error liburing| Project name: liburing
58.35 npm error liburing| Project version: 2.5
58.35 npm error liburing| C compiler for the host machine: cc (gcc 15.2.0 "cc (Alpine 15.2.0) 15.2.0")
58.35 npm error liburing| C linker for the host machine: cc ld.bfd 2.45.1
58.35 npm error liburing| C++ compiler for the host machine: c++ (gcc 15.2.0 "c++ (Alpine 15.2.0) 15.2.0")
58.35 npm error liburing| C++ linker for the host machine: c++ ld.bfd 2.45.1
58.35 npm error liburing| Dependency threads found: YES unknown (cached)
58.35 npm error liburing| Compiler for C supports arguments -Wstringop-overflow: YES
58.35 npm error liburing| Compiler for C supports arguments -Warray-bounds: YES
58.35 npm error liburing| Checking for type "__kernel_rwf_t" : YES
58.35 npm error liburing| Checking whether type "struct __kernel_timespec" has members "tv_sec", "tv_nsec" : YES
58.35 npm error liburing| Checking if "open_how" compiles: YES
58.35 npm error liburing| Checking if "statx" compiles: YES
58.35 npm error liburing| Checking if "glibc_statx" compiles: NO
58.35 npm error liburing| Checking if "C++" compiles: YES
58.35 npm error liburing| Checking for type "ucontext_t" : NO
58.35 npm error liburing| Checking for type "struct nvme_uring_cmd" : YES
58.35 npm error liburing| Has header "sys/fanotify.h" : YES
58.35 npm error liburing| Configuring config-host.h using configuration
58.35 npm error liburing| Configuring compat.h using configuration
58.35 npm error liburing| Configuring io_uring_version.h using configuration
58.35 npm error liburing| Build targets in project: 37
58.35 npm error liburing| Subproject liburing finished.
58.35 npm error
58.35 npm error Build targets in project: 44
58.35 npm error
58.35 npm error liburing 2.5
58.35 npm error
58.35 npm error   Directories
58.35 npm error     bindir  : .
58.35 npm error     libdir  : .
58.35 npm error     datadir : share
58.35 npm error
58.35 npm error   Configuration
58.35 npm error     examples: NO
58.35 npm error     tests   : NO
58.35 npm error
58.35 npm error mediasoup-worker undefined
58.35 npm error
58.35 npm error   Subprojects
58.35 npm error     abseil-cpp : YES
58.35 npm error     catch2     : YES
58.35 npm error     flatbuffers: YES
58.35 npm error     libsrtp3   : YES
58.35 npm error     liburing   : YES
58.35 npm error     libuv      : YES
58.35 npm error     openssl    : YES
58.35 npm error     usrsctp    : YES
58.35 npm error
58.35 npm error   User defined options
58.35 npm error     b_ndebug   : true
58.35 npm error     bindir     :
58.35 npm error     buildtype  : release
58.35 npm error     libdir     :
58.35 npm error     prefix     : /server/node_modules/mediasoup/worker/out/Release
58.35 npm error
58.35 npm error Found ninja-1.10.2.git.kitware.jobserver-1 at /server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/ninja
58.35 npm error "/server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/meson" compile -C "/server/node_modules/mediasoup/worker/out/Release/build" flatbuffers-generator
58.35 npm error ninja: Entering directory `/server/node_modules/mediasoup/worker/out/Release/build'
58.35 npm error [1/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_util.cpp.o
58.35 npm error [2/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_idl_gen_binary.cpp.o
58.35 npm error [3/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_code_generators.cpp.o
58.35 npm error [4/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_file_binary_writer.cpp.o
58.35 npm error [5/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libpython_generator.a.p/grpc_src_compiler_python_generator.cc.o
58.35 npm error [6/49] Linking static target subprojects/flatbuffers-24.3.25/libpython_generator.a
58.35 npm error [7/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_file_name_saving_file_manager.cpp.o
58.35 npm error [8/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libcpp_generator.a.p/grpc_src_compiler_cpp_generator.cc.o
58.35 npm error [9/49] Linking static target subprojects/flatbuffers-24.3.25/libcpp_generator.a
58.35 npm error [10/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libgo_generator.a.p/grpc_src_compiler_go_generator.cc.o
58.35 npm error [11/49] Linking static target subprojects/flatbuffers-24.3.25/libgo_generator.a
58.35 npm error [12/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libts_generator.a.p/grpc_src_compiler_ts_generator.cc.o
58.35 npm error [13/49] Linking static target subprojects/flatbuffers-24.3.25/libts_generator.a
58.35 npm error [14/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_reflection.cpp.o
58.35 npm error [15/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_file_writer.cpp.o
58.35 npm error [16/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libswift_generator.a.p/grpc_src_compiler_swift_generator.cc.o
58.35 npm error [17/49] Linking static target subprojects/flatbuffers-24.3.25/libswift_generator.a
58.35 npm error [18/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_annotated_binary_text_gen.cpp.o
58.35 npm error [19/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_bfbs_gen_lua.cpp.o
58.35 npm error [20/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libjava_generator.a.p/grpc_src_compiler_java_generator.cc.o
58.35 npm error [21/49] Linking static target subprojects/flatbuffers-24.3.25/libjava_generator.a
58.35 npm error [22/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_idl_gen_text.cpp.o
58.35 npm error [23/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_flatc_main.cpp.o
58.35 npm error [24/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_bfbs_gen_nim.cpp.o
58.35 npm error [25/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatc_library.a.p/src_flatc.cpp.o
58.35 npm error [26/49] Linking static target subprojects/flatbuffers-24.3.25/libflatc_library.a
58.35 npm error [27/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_grpc.cpp.o
58.35 npm error [28/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_json_schema.cpp.o
58.35 npm error [29/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_fbs.cpp.o
58.35 npm error [30/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_binary_annotator.cpp.o
58.35 npm error [31/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_php.cpp.o
58.35 npm error [32/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_go.cpp.o
58.35 npm error [33/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_util.cpp.o
58.35 npm error [34/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_lobster.cpp.o
58.35 npm error [35/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_kotlin.cpp.o
58.35 npm error [36/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_dart.cpp.o
58.35 npm error [37/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_text.cpp.o
58.35 npm error [38/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_kotlin_kmp.cpp.o
58.35 npm error [39/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_csharp.cpp.o
58.35 npm error [40/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_python.cpp.o
58.35 npm error [41/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_swift.cpp.o
58.35 npm error [42/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_java.cpp.o
58.35 npm error [43/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_cpp.cpp.o
58.35 npm error [44/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_ts.cpp.o
58.35 npm error [45/49] Compiling C++ object subprojects/flatbuffers-24.3.25/flatc.p/src_idl_gen_rust.cpp.o
58.35 npm error [46/49] Compiling C++ object subprojects/flatbuffers-24.3.25/libflatbuffers.a.p/src_idl_parser.cpp.o
58.35 npm error [47/49] Linking static target subprojects/flatbuffers-24.3.25/libflatbuffers.a
58.35 npm error [48/49] Linking target subprojects/flatbuffers-24.3.25/flatc
58.35 npm error [49/49] Generating fbs/flatbuffers-generator with a custom command
58.35 npm error INFO: autodetecting backend as ninja
58.35 npm error INFO: calculating backend command to run: /server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/ninja -C /server/node_modules/mediasoup/worker/out/Release/build fbs/FBS
58.35 npm error "/server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/meson" compile -C "/server/node_modules/mediasoup/worker/out/Release/build" -j 14 mediasoup-worker
58.35 npm error ninja: Entering directory `/server/node_modules/mediasoup/worker/out/Release/build'
58.35 npm error [1/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_SsnTsnResetRequestParameter.cpp.o
58.35 npm error [2/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_MissingMandatoryParameterErrorCause.cpp.o
58.35 npm error [3/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_SupportedExtensionsParameter.cpp.o
58.35 npm error [4/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_OutgoingSsnResetRequestParameter.cpp.o
58.35 npm error [5/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_UnknownParameter.cpp.o
58.35 npm error [6/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_InvalidStreamIdentifierErrorCause.cpp.o
58.35 npm error [7/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_IncomingSsnResetRequestParameter.cpp.o
58.35 npm error [8/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_ErrorCause.cpp.o
58.35 npm error [9/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_ZeroChecksumAcceptableParameter.cpp.o
58.35 npm error [10/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_ForwardTsnSupportedParameter.cpp.o
58.35 npm error [11/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_UnrecognizedChunkTypeErrorCause.cpp.o
58.35 npm error [12/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_AddOutgoingStreamsRequestParameter.cpp.o
58.35 npm error [13/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_AddIncomingStreamsRequestParameter.cpp.o
58.35 npm error [14/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_ReconfigurationResponseParameter.cpp.o
58.35 npm error [15/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_StaleCookieErrorCause.cpp.o
58.35 npm error [16/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_UnresolvableAddressErrorCause.cpp.o
58.35 npm error [17/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_OutOfResourceErrorCause.cpp.o
58.35 npm error [18/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_RestartOfAnAssociationWithNewAddressesErrorCause.cpp.o
58.35 npm error [19/1401] Compiling C++ object mediasoup-worker.p/src_main.cpp.o
58.35 npm error [20/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_CookieReceivedWhileShuttingDownErrorCause.cpp.o
58.35 npm error [21/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_NoUserDataErrorCause.cpp.o
58.35 npm error [22/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_UnrecognizedParametersErrorCause.cpp.o
58.35 npm error [23/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_InvalidMandatoryParameterErrorCause.cpp.o
58.35 npm error [24/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_ProtocolViolationErrorCause.cpp.o
58.35 npm error [25/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_UserInitiatedAbortErrorCause.cpp.o
58.35 npm error [26/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_errorCauses_UnknownErrorCause.cpp.o
58.35 npm error [27/1401] Compiling C++ object mediasoup-worker.p/src_DepLibUring.cpp.o
58.35 npm error In file included from ../../../src/DepLibUring.cpp:5:
58.35 npm error ../../../src/DepLibUring.cpp: In constructor 'DepLibUring::LibUring::LibUring()':
58.35 npm error ../../../include/Logger.hpp:242:101: warning: format '%lu' expects argument of type 'long unsigned int', but argument 6 has type 'rlim_t' {aka 'long long unsigned int'} [-Wformat=]
58.35 npm error   242 |                         const int loggerWritten = std::snprintf(Logger::buffer, Logger::BufferSize, "W" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
58.35 npm error ../../../src/DepLibUring.cpp:359:33: note: in expansion of macro 'MS_WARN_TAG'
58.35 npm error   359 |                                 MS_WARN_TAG(
58.35 npm error       |                                 ^~~~~~~~~~~
58.35 npm error ../../../src/DepLibUring.cpp:361:105: note: format string is defined here
58.35 npm error   361 |                                   "io_uring_register_buffers() failed due to low RLIMIT_MEMLOCK (soft:%lu, hard:%lu), disabling zero copy: %s",
58.35 npm error       |                                                                                                       ~~^
58.35 npm error       |                                                                                                         |
58.35 npm error       |                                                                                                         long unsigned int
58.35 npm error       |                                                                                                       %llu
58.35 npm error ../../../include/Logger.hpp:242:101: warning: format '%lu' expects argument of type 'long unsigned int', but argument 7 has type 'rlim_t' {aka 'long long unsigned int'} [-Wformat=]
58.35 npm error   242 |                         const int loggerWritten = std::snprintf(Logger::buffer, Logger::BufferSize, "W" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
58.35 npm error ../../../src/DepLibUring.cpp:359:33: note: in expansion of macro 'MS_WARN_TAG'
58.35 npm error   359 |                                 MS_WARN_TAG(
58.35 npm error       |                                 ^~~~~~~~~~~
58.35 npm error ../../../src/DepLibUring.cpp:361:115: note: format string is defined here
58.35 npm error   361 |                                   "io_uring_register_buffers() failed due to low RLIMIT_MEMLOCK (soft:%lu, hard:%lu), disabling zero copy: %s",
58.35 npm error       |                                                                                                                 ~~^
58.35 npm error       |                                                                                                                   |
58.35 npm error       |                                                                                                                   long unsigned int
58.35 npm error       |                                                                                                                 %llu
58.35 npm error [28/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TransportTuple.cpp.o
58.35 npm error [29/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TrendCalculator.cpp.o
58.35 npm error [30/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_Opus.cpp.o
58.35 npm error [31/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_H264.cpp.o
58.35 npm error [32/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_AV1.cpp.o
58.35 npm error [33/1401] Compiling C++ object mediasoup-worker.p/src_RTC_UdpSocket.cpp.o
58.35 npm error [34/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_VP9.cpp.o
58.35 npm error [35/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_VP8.cpp.o
58.35 npm error [36/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtcpFeedback.cpp.o
58.35 npm error [37/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Codecs_DependencyDescriptor.cpp.o
58.35 npm error [38/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtcpParameters.cpp.o
58.35 npm error [39/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpHeaderExtensionUri.cpp.o
58.35 npm error [40/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_Parameters.cpp.o
58.35 npm error [41/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpRtxParameters.cpp.o
58.35 npm error [42/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpCodecParameters.cpp.o
58.35 npm error [43/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpCodecMimeType.cpp.o
58.35 npm error [44/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpHeaderExtensionParameters.cpp.o
58.35 npm error [45/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SctpDictionaries_SctpStreamParameters.cpp.o
58.35 npm error [46/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_Packet.cpp.o
58.35 npm error [47/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_CompoundPacket.cpp.o
58.35 npm error [48/1401] Compiling C++ object mediasoup-worker.p/src_RTC_WebRtcTransport.cpp.o
58.35 npm error [49/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_SenderReport.cpp.o
58.35 npm error [50/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_ReceiverReport.cpp.o
58.35 npm error [51/1401] Compiling C++ object mediasoup-worker.p/src_RTC_WebRtcServer.cpp.o
58.35 npm error [52/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_Bye.cpp.o
58.35 npm error [53/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpParameters.cpp.o
58.35 npm error [54/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_Sdes.cpp.o
58.35 npm error [55/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpNack.cpp.o
58.35 npm error [56/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpSrReq.cpp.o
58.35 npm error [57/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpTmmb.cpp.o
58.35 npm error [58/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPs.cpp.o
58.35 npm error [59/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtp.cpp.o
58.35 npm error [60/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpTllei.cpp.o
58.35 npm error [61/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_Feedback.cpp.o
58.35 npm error [62/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpEcn.cpp.o
58.35 npm error [63/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsPli.cpp.o
58.35 npm error [64/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpEncodingParameters.cpp.o
58.35 npm error [65/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsSli.cpp.o
58.35 npm error [66/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsRpsi.cpp.o
58.35 npm error [67/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsFir.cpp.o
58.35 npm error [68/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsTst.cpp.o
58.35 npm error [69/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackRtpTransport.cpp.o
58.35 npm error [70/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsVbcm.cpp.o
58.35 npm error [71/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsLei.cpp.o
58.35 npm error [72/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsAfb.cpp.o
58.35 npm error [73/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_XrReceiverReferenceTime.cpp.o
58.35 npm error [74/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_XR.cpp.o
58.35 npm error [75/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_FeedbackPsRemb.cpp.o
58.35 npm error [76/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RTCP_XrDelaySinceLastRr.cpp.o
58.35 npm error [77/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_association_NegotiatedCapabilities.cpp.o
58.35 npm error [78/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_association_Socket.cpp.o
58.35 npm error [79/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_association_TransmissionControlBlock.cpp.o
58.35 npm error [80/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_association_StateCookie.cpp.o
58.35 npm error [81/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_TLV.cpp.o
58.35 npm error [82/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_Packet.cpp.o
58.35 npm error [83/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_DataChunk.cpp.o
58.35 npm error [84/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_InitChunk.cpp.o
58.35 npm error [85/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_HeartbeatRequestChunk.cpp.o
58.35 npm error [86/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_HeartbeatAckChunk.cpp.o
58.35 npm error [87/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_InitAckChunk.cpp.o
58.35 npm error [88/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_Chunk.cpp.o
58.35 npm error [89/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_AbortAssociationChunk.cpp.o
58.35 npm error [90/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_SackChunk.cpp.o
58.35 npm error [91/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_ShutdownChunk.cpp.o
58.35 npm error [92/1401] Compiling C++ object mediasoup-worker.p/src_DepLibUV.cpp.o
58.35 npm error [93/1401] Compiling C++ object mediasoup-worker.p/src_DepLibSRTP.cpp.o
58.35 npm error [94/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_ShutdownAckChunk.cpp.o
58.35 npm error [95/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_OperationErrorChunk.cpp.o
58.35 npm error [96/1401] Compiling C++ object mediasoup-worker.p/src_lib.cpp.o
58.35 npm error [97/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_CookieEchoChunk.cpp.o
58.35 npm error [98/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_CookieAckChunk.cpp.o
58.35 npm error [99/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_ShutdownCompleteChunk.cpp.o
58.35 npm error [100/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_ForwardTsnChunk.cpp.o
58.35 npm error [101/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_ReConfigChunk.cpp.o
58.35 npm error [102/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_UnknownChunk.cpp.o
58.35 npm error [103/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_IForwardTsnChunk.cpp.o
58.35 npm error [104/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_Parameter.cpp.o
58.35 npm error [105/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_chunks_IDataChunk.cpp.o
58.35 npm error [106/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_HeartbeatInfoParameter.cpp.o
58.35 npm error [107/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_IPv4AddressParameter.cpp.o
58.35 npm error [108/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_StateCookieParameter.cpp.o
58.35 npm error [109/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_IPv6AddressParameter.cpp.o
58.35 npm error [110/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_UnrecognizedParameterParameter.cpp.o
58.35 npm error [111/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_CookiePreservativeParameter.cpp.o
58.35 npm error [112/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SCTP_packet_parameters_SupportedAddressTypesParameter.cpp.o
58.35 npm error [113/1401] Compiling C++ object mediasoup-worker.p/src_DepLibWebRTC.cpp.o
58.35 npm error [114/1401] Compiling C++ object mediasoup-worker.p/src_DepOpenSSL.cpp.o
58.35 npm error [115/1401] Compiling C++ object mediasoup-worker.p/src_MediaSoupErrors.cpp.o
58.35 npm error [116/1401] Compiling C++ object mediasoup-worker.p/src_Logger.cpp.o
58.35 npm error [117/1401] Compiling C++ object mediasoup-worker.p/src_DepUsrSCTP.cpp.o
58.35 npm error [118/1401] Compiling C++ object mediasoup-worker.p/src_Utils_BitStream.cpp.o
58.35 npm error [119/1401] Compiling C++ object mediasoup-worker.p/src_Utils_File.cpp.o
58.35 npm error [120/1401] Compiling C++ object mediasoup-worker.p/src_Utils_Crypto.cpp.o
58.35 npm error [121/1401] Compiling C++ object mediasoup-worker.p/src_Utils_IP.cpp.o
58.35 npm error [122/1401] Compiling C++ object mediasoup-worker.p/src_Utils_String.cpp.o
58.35 npm error [123/1401] Compiling C++ object mediasoup-worker.p/src_handles_SignalHandle.cpp.o
58.35 npm error [124/1401] Compiling C++ object mediasoup-worker.p/src_Settings.cpp.o
58.35 npm error [125/1401] Compiling C++ object mediasoup-worker.p/src_ChannelMessageRegistrator.cpp.o
58.35 npm error [126/1401] Compiling C++ object mediasoup-worker.p/src_handles_TcpConnectionHandle.cpp.o
58.35 npm error [127/1401] Compiling C++ object mediasoup-worker.p/src_handles_TimerHandle.cpp.o
58.35 npm error [128/1401] Compiling C++ object mediasoup-worker.p/src_handles_TcpServerHandle.cpp.o
58.35 npm error [129/1401] Compiling C++ object mediasoup-worker.p/src_handles_UdpSocketHandle.cpp.o
58.35 npm error [130/1401] Compiling C++ object mediasoup-worker.p/src_handles_UnixStreamSocketHandle.cpp.o
58.35 npm error [131/1401] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelNotifier.cpp.o
58.35 npm error [132/1401] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelNotification.cpp.o
58.35 npm error [133/1401] Compiling C++ object mediasoup-worker.p/src_Worker.cpp.o
58.35 npm error [134/1401] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelSocket.cpp.o
58.35 npm error [135/1401] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelRequest.cpp.o
58.35 npm error [136/1401] Compiling C++ object mediasoup-worker.p/src_RTC_IceCandidate.cpp.o
58.35 npm error [137/1401] Compiling C++ object mediasoup-worker.p/src_RTC_ActiveSpeakerObserver.cpp.o
58.35 npm error [138/1401] Compiling C++ object mediasoup-worker.p/src_RTC_KeyFrameRequestManager.cpp.o
58.35 npm error [139/1401] Compiling C++ object mediasoup-worker.p/src_RTC_DtlsTransport.cpp.o
58.35 npm error ../../../src/RTC/DtlsTransport.cpp: In member function 'bool RTC::DtlsTransport::CheckRemoteFingerprint()':
58.35 npm error ../../../src/RTC/DtlsTransport.cpp:1413:48: warning: writing 1 byte into a region of size 0 [-Wstringop-overflow=]
58.35 npm error  1413 |                 hexFingerprint[(size * 3) - 1] = '\0';
58.35 npm error       |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
58.35 npm error ../../../src/RTC/DtlsTransport.cpp:1348:22: note: at offset 4294967295 into destination object 'hexFingerprint' of size 193
58.35 npm error  1348 |                 char hexFingerprint[(EVP_MAX_MD_SIZE * 3) + 1];
58.35 npm error       |                      ^~~~~~~~~~~~~~
58.35 npm error [140/1401] Compiling C++ object mediasoup-worker.p/src_RTC_NackGenerator.cpp.o
58.35 npm error [141/1401] Compiling C++ object mediasoup-worker.p/src_RTC_IceServer.cpp.o
58.35 npm error [142/1401] Compiling C++ object mediasoup-worker.p/src_RTC_AudioLevelObserver.cpp.o
58.35 npm error [143/1401] Compiling C++ object mediasoup-worker.p/src_RTC_DataProducer.cpp.o
58.35 npm error [144/1401] Compiling C++ object mediasoup-worker.p/src_RTC_DataConsumer.cpp.o
58.35 npm error [145/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Consumer.cpp.o
58.35 npm error [146/1401] Compiling C++ object mediasoup-worker.p/src_RTC_DirectTransport.cpp.o
58.35 npm error [147/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RateCalculator.cpp.o
58.35 npm error [148/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtcLogger.cpp.o
58.35 npm error [149/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpObserver.cpp.o
58.35 npm error [150/1401] Compiling C++ object mediasoup-worker.p/src_RTC_PortManager.cpp.o
58.35 npm error [151/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpProbationGenerator.cpp.o
58.35 npm error [152/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpRetransmissionBuffer.cpp.o
58.35 npm error [153/1401] Compiling C++ object mediasoup-worker.p/src_RTC_PipeTransport.cpp.o
58.35 npm error [154/1401] Compiling C++ object mediasoup-worker.p/src_RTC_PipeConsumer.cpp.o
58.35 npm error [155/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpListener.cpp.o
58.35 npm error [156/1401] Compiling C++ object mediasoup-worker.p/src_RTC_PlainTransport.cpp.o
58.35 npm error [157/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpPacket.cpp.o
58.35 npm error [158/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpStream.cpp.o
58.35 npm error [159/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtxStream.cpp.o
58.35 npm error [160/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SctpListener.cpp.o
58.35 npm error [161/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Producer.cpp.o
58.35 npm error [162/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Serializable.cpp.o
58.35 npm error [163/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SenderBandwidthEstimator.cpp.o
58.35 npm error [164/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpStreamSend.cpp.o
58.35 npm error [165/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Shared.cpp.o
58.35 npm error [166/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SeqManager.cpp.o
58.35 npm error [167/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SctpAssociation.cpp.o
58.35 npm error [168/1401] Compiling C++ object mediasoup-worker.p/src_RTC_RtpStreamRecv.cpp.o
58.35 npm error [169/1401] Compiling C++ object deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_units_data_size.cc.o
58.35 npm error [170/1401] Compiling C++ object deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_units_data_rate.cc.o
58.35 npm error [171/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SharedRtpPacket.cpp.o
58.35 npm error [172/1401] Compiling C++ object deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_network_state_predictor.cc.o
58.35 npm error FAILED: deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_network_state_predictor.cc.o
58.35 npm error c++ -Ideps/libwebrtc/liblibwebrtc.a.p -Ideps/libwebrtc -I../../../deps/libwebrtc -I../../../include -Ifbs -I../../../fbs -I../../../deps/libwebrtc/libwebrtc -I../../../subprojects/openssl-3.0.8/include -I../../../subprojects/openssl-3.0.8/crypto -I../../../subprojects/openssl-3.0.8/crypto/modes -I../../../subprojects/openssl-3.0.8/crypto/ec/curve448 -I../../../subprojects/openssl-3.0.8/crypto/ec/curve448/arch_32 -I../../../subprojects/openssl-3.0.8/providers/common/include -I../../../subprojects/openssl-3.0.8/providers/implementations/include -Isubprojects/openssl-3.0.8/generated-config/archs/linux-aarch64/asm -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-aarch64/asm -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-aarch64/asm/include -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-aarch64/asm/crypto -I../../../subprojects/openssl-3.0.8/generated-config/archs/linux-aarch64/asm/providers/common/include -Isubprojects/abseil-cpp-20240722.0 -I../../../subprojects/abseil-cpp-20240722.0 -I../../../subprojects/flatbuffers-24.3.25/include -I../../../subprojects/flatbuffers-24.3.25/grpc -I../../../subprojects/libuv-v1.51.0/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++17 -O3 -DFLATBUFFERS_LOCALE_INDEPENDENT=false -fPIC -pthread -DMS_LITTLE_ENDIAN -MD -MQ deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_network_state_predictor.cc.o -MF deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_network_state_predictor.cc.o.d -o deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_network_state_predictor.cc.o -c ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.cc
58.35 npm error In file included from ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.cc:2:
58.35 npm error ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:46:33: error: 'int64_t' has not been declared
58.35 npm error    46 |   virtual BandwidthUsage Update(int64_t send_time_ms,
58.35 npm error       |                                 ^~~~~~~
58.35 npm error ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:17:1: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
58.35 npm error    16 | #include <vector>
58.35 npm error   +++ |+#include <cstdint>
58.35 npm error    17 |
58.35 npm error ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:47:33: error: 'int64_t' has not been declared
58.35 npm error    47 |                                 int64_t arrival_time_ms,
58.35 npm error       |                                 ^~~~~~~
58.35 npm error ../../../deps/libwebrtc/libwebrtc/api/network_state_predictor.h:47:33: note: 'int64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
58.35 npm error [173/1401] Compiling C++ object deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_modules_pacing_interval_budget.cc.o
58.35 npm error [174/1401] Compiling C++ object deps/libwebrtc/liblibwebrtc.a.p/libwebrtc_api_units_frequency.cc.o
58.35 npm error [175/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Router.cpp.o
58.35 npm error [176/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SrtpSession.cpp.o
58.35 npm error [177/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TcpConnection.cpp.o
58.35 npm error [178/1401] Compiling C++ object mediasoup-worker.p/src_RTC_StunPacket.cpp.o
58.35 npm error [179/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TcpServer.cpp.o
58.35 npm error [180/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TransportCongestionControlClient.cpp.o
58.35 npm error [181/1401] Compiling C++ object mediasoup-worker.p/src_RTC_TransportCongestionControlServer.cpp.o
58.35 npm error [182/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SimpleConsumer.cpp.o
58.35 npm error [183/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SvcConsumer.cpp.o
58.35 npm error [184/1401] Compiling C++ object mediasoup-worker.p/src_RTC_SimulcastConsumer.cpp.o
58.35 npm error [185/1401] Compiling C++ object mediasoup-worker.p/src_RTC_Transport.cpp.o
58.35 npm error ninja: build stopped: subcommand failed.
58.35 npm error INFO: autodetecting backend as ninja
58.35 npm error INFO: calculating backend command to run: /server/node_modules/mediasoup/worker/out/pip_meson_ninja/bin/ninja -C /server/node_modules/mediasoup/worker/out/Release/build -j 14 mediasoup-worker
58.35 npm error npm-scripts.mjs [ERROR] [postinstall] downloadPrebuiltWorker() | fetched mediasoup-worker prebuilt binary fails to run in this host [status:127]
58.35 npm error WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
58.35 npm error WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
58.35 npm error WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
58.35 npm error npm-scripts.mjs [ERROR] [postinstall] executeCmd() failed, exiting: Error: Command failed: "python3" -m invoke -r worker mediasoup-worker
58.35 npm error A complete log of this run can be found in: /root/.npm/_logs/2025-12-12T00_27_13_430Z-debug-0.log
------

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@
 
 - `RtxStream`: Don't check if RTP timestamp moved backwards ([PR #1668](https://github.com/versatica/mediasoup/pull/1668), credits to @Lynnworld).
 - Fix RTX packets containing non yet seen RTP packets being discarded ([PR #1653](https://github.com/versatica/mediasoup/pull/1653), credits to @penguinol, @mengbieting and @Lynnworld).
+- Worker: Fix missing system header include, which fails in GCC 15.
 
 ### 3.19.12
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,7 +6,7 @@
 
 - `RtxStream`: Don't check if RTP timestamp moved backwards ([PR #1668](https://github.com/versatica/mediasoup/pull/1668), credits to @Lynnworld).
 - Fix RTX packets containing non yet seen RTP packets being discarded ([PR #1653](https://github.com/versatica/mediasoup/pull/1653), credits to @penguinol, @mengbieting and @Lynnworld).
-- Worker: Fix missing system header include, which fails in GCC 15.
+- Worker: Fix missing system header include, which fails in GCC 15 ([PR #1679](https://github.com/versatica/mediasoup/pull/1679), credits to @upisfree).
 
 ### 3.19.12
 

--- a/worker/deps/libwebrtc/libwebrtc/api/network_state_predictor.h
+++ b/worker/deps/libwebrtc/libwebrtc/api/network_state_predictor.h
@@ -14,6 +14,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 namespace webrtc {
 

